package.json

```diff
@@ -54,6 +57,7 @@
     "jest": "^26.6.3",
     "lint-staged": "^10.5.4",
     "optimist": "^0.6.1",
+    "parcel": "^1.12.4",
```

Comment:
💅🏻  We could use `esbuild` instead, should be faster too.

Reply:
Swithed to esbuild, just to note that ES6 it's the earliest it can compile to, but I think we're fine with support:

- Chrome 49+
- Safari 10.1+
- Firefox 45+
- Edge 14+